projects/jupyter-collaboration/pyproject.toml

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+[build-system]
+build-backend = "hatchling.build"
+requires = ["hatchling>=1.4.0"]
+
+[project]
+name = "jupyter-collaboration"
+readme = "README.md"
+license = { file = "LICENSE" }
+requires-python = ">=3.8"
+description = "JupyterLab/Jupyter Notebook 7+ Real Time Collaboration extension (metapackage)"
+classifiers = [
+    "Intended Audience :: Developers",
+    "Intended Audience :: System Administrators",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: BSD License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Framework :: Jupyter",
+    "Framework :: Jupyter :: JupyterLab",
+    "Framework :: Jupyter :: JupyterLab :: 4",
+    "Framework :: Jupyter :: JupyterLab :: Extensions",
+    "Framework :: Jupyter :: JupyterLab :: Extensions :: Prebuilt",
+]
+dependencies = [
+    "jupyter-server-ydoc",
+    "jupyter-collaboration-ui",
+    "jupyter-docprovider"
```

Comment:
Should we start pinning dependencies? At least we shouldn't install `jupyter-server-ydoc` v0.8.0 which stills exist on [PyPI](https://pypi.org/project/jupyter-server-ydoc).

Reply:
Good point, thanks. I set their versions to 1.0 alpha 0 and pinned here to `>=1.0.0a0`.